exec/java-exec/src/main/java/org/apache/drill/exec/rpc/user/UserSession.java

```diff
@@ -59,9 +62,11 @@
   public static final String USER = "user";
   public static final String PASSWORD = "password";
   public static final String IMPERSONATION_TARGET = "impersonation_target";
+  public static final String QUOTING_IDENTIFIERS_CHARACTER = "quoting_identifiers_character";
```

Comment:
isn't this a bit too long? could "quoting" be used instead? (also, brackets are two characters... :) )